docs/user/InstructionsForAutoTuner.md

```diff
@@ -193,6 +193,17 @@ We show three different views possible at the end, namely: `Table View`, `Scatte
 ![Parallel Coordinate View](../images/Autotuner_best_parameter_view.webp)
 <p style="text-align: center;">Parallel Coordinate View (best run is in green)</p>
 
+## Testing framework
+
+```
+# Activate your virtualenv before this
+conda activate autotuner_env
+
+# Smoke test
+cd tools/AutoTuner/tests
+python3 smoke_test_tune.py
+python3 smoke_test_sweep.py
```

Comment:
Could the tests themselves call the `setup.sh` (as suggested above) to activate the environment?

Reply:
They can, but from the same principles of ORFS can we leave it optional? Similar to how we have `Env.sh` as optional

Reply:
Update the docs here. 3 step instructions
- install depen
- source env
- run test.